android/package.json

```diff
@@ -17,11 +17,11 @@
 		"integrity": "sha512-A0tV+fYtkpKfIF5roRTCFPtdULMFygmfWlEuuHOBjC3q4rz/mKnAsJTYBlqayC/4oYEWehj867Oh1o6vy26XHQ=="
 	},
 	"minSDKVersion": "21",
-	"compileSDKVersion": "31",
+	"compileSDKVersion": "33",
 	"vendorDependencies": {
-		"android sdk": ">=23.x <=31.x",
-		"android build tools": ">=30.0.2 <=31.x",
-		"android platform tools": "31.x",
+		"android sdk": ">=23.x <=33.x",
+		"android build tools": ">=30.0.2 <=33.x",
+		"android platform tools": "33.x",
```

Comment:
Did we really skip 32?!

Reply:
32 is [Android 12L](https://de.wikipedia.org/wiki/Android_12L)